app/javascript/controllers/copy_controller.js

```diff
@@ -1,4 +1,3 @@
-import bootstrap from 'bootstrap/dist/js/bootstrap'
```

Comment:
don't think we need this line?  it causes a warning in the js console and isn't in any other stimulus controller